### PR TITLE
New package: Biopandas python package

### PR DESCRIPTION
**Pull request by ilbiondo**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-biopandas/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    git      = "https://github.com/rasbt/biopandas.git"
+
+    # Note that the source package on PyPi is broken as it
+    # is missing the requirements.txt so we have to download
+    # from github
+
+    version('0.2.5', branch="v0.2.5")
+
+    depends_on('python@3.5:',       type=('build', 'run'))
+    depends_on('py-pandas@0.19.1:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-biopandas/package.py]:
```suggestion
    depends_on('py-pandas@0.24.2:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-biopandas/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-numpy@1.11.2:',  type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-biopandas/package.py]:
```suggestion
    depends_on('py-numpy@1.16.2:',  type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-biopandas/package.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-scipy@0.18.1:',  type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-biopandas/package.py]: I don't see this in the `requirements.txt`

**Comment by ilbiondo** [comment]
The py-scipy requirement (and the original versions) came from the documentation:

See "requirements" at http://rasbt.github.io/biopandas/installation/

I've checked the code and have to agree that py-scipy probably isn't needed and quick check indicates that biopandas functions correctly with out it.

**Comment by adamjstewart** [comment]
Yeah, the documentation and README seem pretty out-of-date. If you grep the git repo, there's no usage of scipy in any file.

**Pull request merged**